wolfcrypt/benchmark/benchmark.c

```diff
@@ -314,6 +319,31 @@
 #define EXIT_FAILURE 1
 #endif
 
+#undef LIBCALL_CHECK_RET
```

Comment:
This is duplicated in test.h. I realize we don't include test.h here for portability reasons. Might make sense to put in the wolfssl/wolfcrypt/types.h or just add a new header for things like this... or change test.h to have some ifdef logic to gate out the non-portable socket stuff.

Reply:
the macro definitions aren't exactly the same.  and btw the first thing I tried was `#include <wolfssl/test.h>` in benchmark.c, which went.... poorly.  I decided to not let the perfect be the enemy of the good.